app/src/main/kotlin/io/privkey/keep/storage/AndroidKeystoreStorage.kt

```
@@ -229,4 +262,182 @@ class AndroidKeystoreStorage(private val context: Context) : SecureStorage {
+        val sharePrefs = getSharePrefs(key)
+        val cleared = sharePrefs.edit().clear().commit()
+        if (!cleared) {
+            throw KeepMobileException.StorageException("Failed to clear share metadata")
+        }
+
+        val alias = getKeystoreAlias(key)
+        try {
+            if (keyStore.containsAlias(alias)) {
+                keyStore.deleteEntry(alias)
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)